python/src/aiconfig/editor/travel.aiconfig.json

```diff
@@ -115,6 +115,15 @@
         "parameters": {}
       },
       "outputs": []
+    },
+    {
+      "name": "prompt_5",
+      "input": "",
+      "metadata": {
+        "model": "gpt-4",
+        "parameters": {}
+      },
+      "outputs": []
```

Comment:
Intentional?

Reply:
Whoops, no. Thanks for catching, will revert this change